### PR TITLE
chore(Auth/Logs): update logging info

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Description
- Update info after new logging system

## Related Issue
-

### CODE REVIEW

File: docs/logs.md [hunk 1]
```diff
@@ -34,34 +38,20 @@ The [RealmID](realmlist#id) of the realm this log string came from. 0 if realmd.
...
-| LOG_TYPE_BASIC  | 2     | BASIC INFORMATION           |
-| LOG_TYPE_DETAIL | 3     | DETAILED INFORMATION        |
-| LOG_TYPE_DEBUG  | 4     | DEBUGGING INFORMATION       |
-| LOG_TYPE_CHAR   | 5     | CHARACTER-RELATED           |
-| LOG_TYPE_WORLD  | 6     | WORLD PACKET DUMP           |
-| LOG_TYPE_RA     | 7     | REMOTE ACCESS LOGS          |
-| LOG_TYPE_GM     | 8     | GM COMMAND LOGS             |
-| LOG_TYPE_CRASH  | 9     | CRASH ALARM LOGS            |
-| LOG_TYPE_CHAT   | 10    | CHAT/MESSAGE LOGS           |
+The `Logger` name
```
Comment (Helias) [inline docs/logs.md:41]: any example..?

Comment (Winfidonarleyan) [reply docs/logs.md]: added

**Review by Helias — APPROVED** [review]

File: docs/logs.md [hunk 2]
```diff
@@ -4,7 +4,11 @@
 
 **The \`logs\` table**
 
-This table stores all logs from authserver if database logging in config is enabled.
+This table stores logs from `Apennder` type database in config file.
```
Comment (Kitzunu) [inline docs/logs.md:7]: Misspelling on appender

Comment (Winfidonarleyan) [reply docs/logs.md]: good

File: docs/logs.md [hunk 3]
```diff
@@ -4,7 +4,11 @@
 
 **The \`logs\` table**
 
-This table stores all logs from authserver if database logging in config is enabled.
+This table stores logs from `Appennder` type database in config file.
```
Comment (Kitzunu) [inline docs/logs.md:7]: Appender :D

Comment (Winfidonarleyan) [reply docs/logs.md]: Ahah

**Pull request merged**